### PR TITLE
Updated to work with most recent version of Akka.Persistence

### PR DESCRIPTION
**Pull request by alextobias48**

### CODE REVIEW

**Comment by Danthar** [comment]
@Aaronontheweb Do we have docker CI integration for Cassandra ? (build config in TC is currently paused)

**Comment by Aaronontheweb** [comment]
@Danthar not at the moment; should have it online in a couple of weeks

File: src/Akka.Persistence.Cassandra/Akka.Persistence.Cassandra.csproj [hunk 1]
```diff
@@ -32,32 +32,41 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
-    <Reference Include="Akka, Version=1.0.6.16, Culture=neutral, processorArchitecture=MSIL">
-      <HintPath>..\packages\Akka.1.0.6\lib\net45\Akka.dll</HintPath>
+    <Reference Include="Akka, Version=1.1.2.29, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>..\..\..\..\..\..\..\Sandhills\Nuget\Akka.1.1.2\lib\net45\Akka.dll</HintPath>
```
Comment (DavidHamburg) [inline src/Akka.Persistence.Cassandra/Akka.Persistence.Cassandra.csproj:36]: This path should be ..\packages\Akka.1.1.2\lib\net45\Akka.dll

**Comment by Danthar** [comment]
Also, there are multiple PR's now that supposedly update this repo. For example does this PR also cover everything from https://github.com/akkadotnet/Akka.Persistence.Cassandra/pull/23 and what about https://github.com/akkadotnet/Akka.Persistence.Cassandra/pull/30 ?

**Comment by Aaronontheweb** [comment]
Merged this so @heynickc can get the CI up and running for Akka.Persistence.Cassandra, FYI

**Pull request merged**